sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -637,6 +637,15 @@ object RapidsConf {
       .booleanConf
       .createWithDefault(false)
 
+  val ENABLE_CREATE_MAP = conf("spark.rapids.sql.createMap.enabled")
+    .doc("When set to true, support the CreateMap expression on the GPU with multiple " +
```

Comment:
nit: Could we try to reword this a bit? I'm not sure we need all of the details, but I am fine if others disagree.

```
The GPU enabled version of the CreateMap expression, `map` SQL function, does not detect duplicate keys in all 
cases and does not guarantee which key wins if there are duplicates. When this config is set to true `CreateMap`
will be enabled to run on the GPU even in cases where there might be duplicate keys.
```